### PR TITLE
Convert site to Flask

### PR DESCRIPTION
**Pull request by nottrobin**
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2145, for https://github.com/canonical-web-and-design/web-squad/issues/2148

QA
--

- `./run clean && ./run`, go to `http://localhost:8026` and see the site, check it works
- `./run build && docker build -t multipass . && docker run -ti -p 8126:80 multipass`, then go to http://localhost:8126, see the site
- `./run test`, check it works
- check the CI tests pass

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
-# Multipass
-[![CircleCI build status](https://circleci.com/gh/canonical-web-and-design/multipass.run.svg?style=shield)](https://circleci.com/gh/canonical-web-and-design/multipass.run) [![Code coverage](https://codecov.io/gh/canonical-web-and-design/multipass.run/branch/master/graph/badge.svg)](https://codecov.io/gh/canonical-web-and-design/multipass.run)
+# Canonical Webteam Website-Boilerplate
 
-Codebase for https://multipass.run - the marketing website for Canonical's multipass product.
+[![CircleCI build status](https://circleci.com/gh/canonical-web-and-design/canonicalwebteam.website-boilerplate.svg?style=shield)](https://circleci.com/gh/canonical-web-and-design/canonicalwebteam.website-boilerplate) [![Code coverage](https://codecov.io/gh/canonical-web-and-design/canonicalwebteam.website-boilerplate/branch/master/graph/badge.svg)](https://codecov.io/gh/canonical-web-and-design/canonicalwebteam.website-boilerplate)
```
[the remark this answers is not in the document]
Comment (nottrobin) [reply README.md]: Good catch. I completely forgot about that bit.

[the remark this answers is not in the document]
Comment (nottrobin) [reply README.md]: Fixed.

**Comment by jkfran** [comment]
Looks good to me

File: webapp/app.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from canonicalwebteam.flask_base.app import FlaskBase
+from flask import render_template
+
+# Rename your project below
+app = FlaskBase(
+    __name__,
+    "project-name.com",
```
Comment (jpmartinspt) [inline webapp/app.py]: `multipass.run`

File: webapp/app.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from canonicalwebteam.flask_base.app import FlaskBase
+from flask import render_template
+
+# Rename your project below
```
Comment (jpmartinspt) [inline webapp/app.py]: Comment can be deleted.

**Pull request merged**